Improvement/nearly finished

filipemartinsfe opened the pull request.

Commit:
content size change, media query changed as well

Commit:
requested changes from clemens, added css formatter

Commit:
change roles and added svg sprites

clemensf commented on index.html:
```diff
@@ -234,13 +208,48 @@ <h2 class="h2">subheadline lorem ipsum</h2>
+      <section>
+        <div class="document-group">
+          <h2>All Document Formats</h2>
+          <ul class="doc-type">
```
I would call this document-list

clemensf commented on index.html:
```diff
@@ -234,13 +208,48 @@ <h2 class="h2">subheadline lorem ipsum</h2>
+          <ul class="doc-type">
+            <li> 
+              <img src="img/pdf.svg" alt="">
+              <div class="doc-content">
```
is this wrapper div.doc-content really required?

clemensf commented on index.html:
```diff
@@ -332,10 +332,23 @@ <h4 class="h4">Small Video Name</h4>
           </div>
+          <div class="document-group">
+              <h2>Document List</h2>
+              <ul class="doc-type">
```
see above. here already the headline says "document list" :-)

clemensf commented on css/style.css:
```diff
@@ -980,3 +985,28 @@ teaser .teaser-image {
 }
+
+/********************** SVG MARKUP */
+/*********** SVG MARKUP ************/
```
one comment should be enough :-)

clemensf commented on index.html:
```diff
@@ -332,10 +332,23 @@ <h4 class="h4">Small Video Name</h4>
                   Aliquet bibendum enim facilisis gravida neque convallis.
                 </p>
           </div>
```
separate the three teasers like we talked about it.
wrap them by a "teaser-row" div. 
Also do this for the single teaser and the double teaser

Commit:
some of the changes were made, no comments on github found

Commit:
stage back removed / changed to after on main div
teaser-row added / removed single and multiple teasers
styles changed : font weight on text, changed some structural css to manage the html changes, re-use of some class's in order to make the grid flow work in responsiveness

clemensf commented on css/style.css:
```diff
@@ -202,83 +173,86 @@ h4 {
-  border-radius: 80px;
-  background: url('../img/search.svg') center/cover no-repeat,linear-gradient(rgba(0,0,0,.12),rgba(0,0,0,.12));
+  background: url('../img/search.svg')
+  center/cover no-repeat,linear-gradient(rgba(0,0,0,.12),rgba(0,0,0,.12));
```
I won't wrap lines like that in css files.

clemensf commented on css/style.css:
```diff
@@ -616,12 +551,10 @@ blockquote:after {
 }
 
 .form-group select {
+  -webkit-appearance: none;
```
why  -webkit-appearance: none;?

clemensf commented on css/style.css:
```diff
@@ -632,90 +565,76 @@ blockquote:after {
-
-  background: url(../img/drop-arrow.svg) center/cover no-repeat,linear-gradient(rgba(0,0,0,.12), rgba(0,0,0,.12));
+  background: url(../img/drop-arrow.svg)
+  center/cover no-repeat,linear-gradient(rgba(0,0,0,.12), rgba(0,0,0,.12));
```
see above, I won't wrap line like that.

clemensf commented on index.html:
```diff
@@ -2,367 +2,699 @@
+
         <div class="contacts-group">
           <div class="contacts-image">
-              <img src="https://dummyimage.com/480x240/e9e9e9/ccccccc" alt="" class="img-responsive" >
```
why did you use a landscape picture for a portrait picture?

clemensf commented on css/style.css:
```diff
@@ -594,7 +603,7 @@ teaser .teaser-image{
 /*********** Media Query ************/
 /* Media Query **********************/
```
remove duplicate comment :-)

clemensf commented on css/style.css:
```diff
@@ -594,7 +603,7 @@ teaser .teaser-image{
-@media(min-width:600px){
+@media(min-width:1024px){
   .breadcrumbs {
     grid-column:span 3;
```
why is this required? It works also with grid-column: 1/4 which is already set before.

Commit:
commented old teaser modules, added new module of teaser

Commit:
basic f1
